public/login.js

```diff
@@ -23,4 +44,6 @@ window.addEventListener('DOMContentLoaded', () => {
   form.addEventListener('submit', login)
 })
 
-window.addEventListener('login-in-processing', addLoadingAnimation, false)
+window.addEventListener('login-in-processing', addLoadingAnimation)
+window.addEventListener('login-in-finished', removeLoadingAnimation)
```

Comment:
I would handle this in iframe only and to not pass above. What the reason to pass it to the parent app?

Reply:
I tried to explain in it description comment